catalyst/dl/utils/criterion/accuracy.py

```diff
@@ -1,15 +1,29 @@
 import numpy as np
+from catalyst.utils import get_activation_fn
 
 
-def accuracy(outputs, targets, topk=(1, )):
+def accuracy(outputs,
```

Comment:
code style, please

Reply:
@TezRomacH What code style is brocken here? `make codestyle` shows only docstrings warnings. On this page I cannot see specifics rules https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md

Reply:
@Dipet catalyst codestyle
```
def accuracy(
    outputs,
    targets,
    topk=(1, ),
    threshold: float = None,
    activation: str = None
):
```
This format allows your to refactor all your function names more easily.